engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -1331,6 +1334,8 @@ struct ashen_hallow_t : public paladin_spell_t
     hd_damage   = new hallowed_discernment_tick_t( p );
     damage_tick = new ashen_hallow_tick_t( p, hd_damage );
     heal_tick   = new ashen_hallow_heal_tick_t( p );
+    cooldown    = p -> cooldowns.ashen_hallow;
+    cooldown -> duration = p -> covenant.venthyr->cooldown();
```

Comment:
Do you really need that? this should be automagic'd and get_cooldown() will link to the same object as this action's cooldown as long as the action is called the same name as the argument of get_cooldown()